types/prop-types/prop-types-tests.ts

```diff
@@ -20,7 +20,7 @@ interface Props {
     instanceOf: TestClass;
     oneOf: 'a' | 'b' | 'c';
     oneOfType: string | boolean | {
-        foo?: string;
+        foo?: string | null;
```

Comment:
Why did this and `shape.bar` below it need to be changed to both be nullable?

Reply:
To be honest I'm not sure why this wasn't needed before. It came up as an error so I happily patched it, because `| null` is the more correct type (`PropTypes.string` would allow `null` through). I understand that it may not be desirable from a practical standpoint, but maybe the better way to resolve that is to remove `|null` from the `Requireable` type? At least that way it's implemented in a consistent way (notice that the root-level optional type *already* had to include `null`; the change now is that nested types match the same behaviour)